src/ClusterBootstrap/deploy.py

```diff
@@ -2810,6 +2832,10 @@ def start_one_kube_service(fname):
         except Exception as e:
             pass
 
+    if fname == "./deploy/services/jobmanager/jobmanager.yaml":
```

Comment:
Is there any better to create the configman with jobmanager? consider this: if we change the name or path of jobmanager.yaml later, this hard-coded path will be invalided. also when we stop jobmanager, we need to make sure that this configmap should be deleted as well.

Reply:
This is actually not written by me. I think it's Di. I missed this change while editing my own file, so just copied it back.